beacon-chain/rpc/eth/helpers/error_handling.go

```diff
@@ -36,10 +36,14 @@ type SingleIndexedVerificationFailure struct {
 }
 
 // SyncDetails contain details about sync status.
+// TODO: Duplicated in structs.go
+// QUESTION: Why are these structs PascalCased, but those in structs.go are camelCased?
```

Comment:
PascalCased structs are visible to other packages, camelCase are not. This struct is defined in `helpers` package, but we *used* it in other packages. I think at some point there was a refactor and now we use it only internally. So it should be camelCased. It should also be moved to `sync.go` because that's where it's used.